src/test/java/com/o19s/es/ltr/query/ValidatingLtrQueryBuilderTests.java

```diff
@@ -108,7 +108,6 @@ protected boolean builderGeneratesCacheableQueries() {
         return false;
     }
 
-    @Override
```

Comment:
Please see https://github.com/elastic/elasticsearch/pull/35967/commits/e882e624ad89bbaf279035907fda9d91295748d7
supportsBoostAndQueryName has been removed in favor of two distinct methods supportsBoost and supportsQueryName